sqlmesh/core/model/kind.py

```diff
@@ -280,6 +279,18 @@ def _parse_path(cls, v: t.Any) -> str:
             return v.this
         return str(v)
 
+    @field_validator("csv_settings", mode="before")
+    @classmethod
+    def _parse_csv_settings(cls, v: t.Any) -> t.Optional[CsvSettings]:
+        if v is None or isinstance(v, CsvSettings):
+            return v
+        if isinstance(v, exp.Expression):
+            tuple_exp = parse_properties(cls, v, {})
```

Comment:
Should we pass `dialect` in the `values` arg here?

Reply:
Where do we take it from? Additionally it's irrelevant here since we are not parsing a string here neither do we need to store tie dialect in the expresson's meta, since we throw the tuple away.

Reply:
Both points make sense 👍